staff/oriol-bover/e-commerce-template/style.css

```diff
@@ -0,0 +1,178 @@
+body {
+    background:whitesmoke;
+    font-family: sans-serif;
+  }
+
+  .card-img {
+    max-width:100%;
+  }
+
+  .header {
+    position: fixed;
+    top: 0;
+    width: 100%;
+    padding: 1rem;
+    background-color: lightslategray;
+    height: 4rem;
+    box-shadow: 0 3px 5px rgba(57, 63, 72, 0.3);
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .brand_logo {
+    flex-grow: 1;
+  }
+
+  .brand_logo a {
+    margin-left: 0.5rem;
+    font-size: 3rem;
+    color: whitesmoke;
+  }
+
+  .brand_logo a:hover{
+    color: lightgray;
+  }
+
+  .form{
+    flex-grow: 2;
+    margin-left: 1rem;
+  }
+
+  .search_input{
+    padding: 0.5rem;
+    border: 1px solid white;
+    border-radius: 0.5rem;
+  }
+
+  .search_button {
+    padding: 0.5rem;
+    border: 1px solid white;
+    border-radius: 0.5rem;
+    cursor:pointer;
+  }
+
+  .search_button:hover {
+    color: lightslategray;
+  }
+
+  .normal_menu{
+    visibility: hidden;
```

Comment:
Be careful with visibility hidden instead of display none, cause even you don't see the element, it's still there, so it's space it's occupied.

Reply:
Done! thanks for the advice